plugins/mcreator-localization/lang/texts.properties

```diff
@@ -542,6 +542,7 @@ blockly.block.entity_set_sprinting=Set sprinting of %1 to %2
 blockly.block.entity_set_sneaking=Set sneaking of %1 to %2
 blockly.block.entity_get_absorption=Get absorption health level of %1
 blockly.block.entity_inventory_foreach=For each item copy as %2 of inventory of %1 do %3
+blockly.block.world_use_bonemeal=Simulate bonemeal use at x: %1 y: %2 z: %3

```

Comment:
Simulate word is used to describe an action that only attempts to do something, but does not actually do it.

I suggest changing the text so something like "Apply bonemeal action at ..." or something similar like this.